packages/itwinui-css/src/table/table.scss

```diff
@@ -215,22 +183,22 @@
       visibility: visible;
     }
 
-    .iui-row-expander > .iui-button-icon {
+    .iui-table-row-expander > .iui-button-icon {
       @media (prefers-reduced-motion: no-preference) {
         transition: transform $iui-speed-fast ease-out;
       }
     }
 
-    &.iui-row-expanded {
+    &.iui-table-row-expanded {
```

Comment:
Now the question is to what extent we want to prefix with `iui-table-*` 😀 I guess here find replace all did the job. Maybe we should prefix all elements but not modifiers.